src/ComposerJsonRetrievalTrait.php

```diff
@@ -11,11 +11,11 @@
 use Composer\Factory;
 use Composer\Json\JsonFile;
 
+use function call_user_func;
```

Comment:
Since this patch updates the skeleton to set the minimum PHP version to 7.3, we can stop using `call_user_func()`, and instead just call the callable directly (as 7.2 forwards allows any callable to be directly invoked). Please remove this import, and then update the `getComposerJson()` call such that:

```php
$composerFile = call_user_fun($this->composerFileFactory);
```

becomes:

```php
$composerFile = ($this->composerFileFactory)();
```

Reply:
Done